rosbag2_examples/rosbag2_examples_py/rosbag2_examples_py/simple_bag_reader.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2024 Sony Group Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+
+import rclpy
+from rclpy.executors import ExternalShutdownException
+from rclpy.node import Node
+import rosbag2_py
+from std_msgs.msg import String
+
+
+class SimpleBagReader(Node):
+
+    def __init__(self, bag_filename):
+        super().__init__('simple_bag_reader')
+        self.reader = rosbag2_py.SequentialReader()
+        storage_options = rosbag2_py._storage.StorageOptions(
+            uri=bag_filename,
+            storage_id='sqlite3')
```

Comment:
@fujitatomoya I've double-checked, we actually don't use `storage_id` anywhere in the `SequantialReader` class. It would be better to show for users that it could be empty and don't need to bother to figure out what it is.

Reply:
@MichaelOrlov the thing is all existed examples in `rosbag2_examples_py` use `storage_id='sqlite3'`, so i just keep this aligned with those to make it work.